docs/en/operations/system-tables/errors.md

```diff
@@ -9,11 +9,15 @@ Columns:
 
 - `name` ([String](../../sql-reference/data-types/string.md)) — name of the error (`errorCodeToName`).
 - `code` ([Int32](../../sql-reference/data-types/int-uint.md)) — code number of the error.
-- `value` ([UInt64](../../sql-reference/data-types/int-uint.md)) — the number of times this error has been happened.
-- `last_error_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — time when the last error happened.
+- `value` ([UInt64](../../sql-reference/data-types/int-uint.md)) — the number of times this error happened.
+- `last_error_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — the time when the last error happened.
 - `last_error_message` ([String](../../sql-reference/data-types/string.md)) — message for the last error.
-- `last_error_trace` ([Array(UInt64)](../../sql-reference/data-types/array.md)) — A [stack trace](https://en.wikipedia.org/wiki/Stack_trace) which represents a list of physical addresses where the called methods are stored.
-- `remote` ([UInt8](../../sql-reference/data-types/int-uint.md)) — remote exception (i.e. received during one of the distributed query).
+- `last_error_trace` ([Array(UInt64)](../../sql-reference/data-types/array.md)) — A [stack trace](https://en.wikipedia.org/wiki/Stack_trace) that represents a list of physical addresses where the called methods are stored.
+- `remote` ([UInt8](../../sql-reference/data-types/int-uint.md)) — remote exception (i.e. received during one of the distributed queries).
+
+:::note
```

Comment:
@Algunenano, why do we increment counters in the Exception constructor rather than doing that at the top level of catching them?

Reply:
I'm not sure, but I guess that doing it in the constructor is simpler and impossible to forget. Doing it in the catch block requires remembering to do it and nobody will.